eng/pipelines/fluent_integration.yaml

```diff
@@ -25,28 +25,37 @@ jobs:
     name: azsdk-pool-mms-win-2022-general
     vmImage: windows-2022
 
+  variables:
+    - template: /eng/pipelines/variables/globals.yml
+
   steps:
   - task: NodeTool@0
     inputs:
-      versionSpec: '18.x'
+      versionSpec: '$(NodeVersion)'
 
   - script: npm install autorest -g
 
+  - template: /eng/pipelines/steps/cache-maven-repository.yml
+    parameters:
+      JobType: 'Fluent Integration'
+
   - task: Maven@3
     inputs:
       mavenPomFile: 'pom.xml'
-      mavenOptions: '-Xmx3072m'
+      mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
       javaHomeOption: 'JDKVersion'
-      jdkVersionOption: '1.11'
+      jdkVersionOption: $(JavaVersion)
       jdkArchitectureOption: 'x64'
       publishJUnitResults: true
       testResultsFiles: '**/TEST-*.xml'
       goals: 'clean install'
-      options: '-P local,testFluent --no-transfer-progress'
+      options: '$(DefaultOptions) -P local,testFluent --no-transfer-progress -T 1C'
```

Comment:
curious why do we add `-T 1C`?

Reply:
This was done to parallelize the build. Before this wasn't done as `fluent-test` didn't depend on `fluent-gen` and running this in parallel could start code generation for Fluent tests before the fluent code generator was even compiled.